### PR TITLE
Update OpenAPI 3.0 Specifications

### PR DESCRIPTION
**Pull request by sraj-cloud**
This PR updates the OpenAPI 3.0 specification files with the latest changes.

### CODE REVIEW

**Review by gregjopa — APPROVED** [review]

**Comment by bachhoant** [comment]
ec4d8f36a6ab0abcc3865e1072aaddc85ea14d5c

**Comment by bachhoant** [comment]
fb6f12627e27331d8fb9117a0bc077cef632cc69

**Review by bachhoant — COMMENTED** [review]
fb6f12627e27331d8fb9117a0bc077cef632cc69

File: openapi/invoicing_v2.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
   "info": {
     "title": "Invoices",
     "description": "Use the Invoicing API to create, send, and manage invoices. You can also use the API or webhooks to track invoice payments. When you send an invoice to a customer, the invoice moves from draft to payable state. PayPal then emails the customer a link to the invoice on the PayPal website. Customers with a PayPal account can log in and pay the invoice with PayPal. Alternatively, customers can pay as a guest with a debit card or credit card. For more information, see the <a href=\"/docs/invoicing/\">Invoicing Overview</a> and the <a href=\"/docs/invoicing/basic-integration/\">Invoicing Integration Guide</a>.",
```
Comment (Kov65) [inline openapi/invoicing_v2.json:5]: Pay

**Review by Kov65 — COMMENTED** [review]
Pay

File: openapi/invoicing_v2.json (hunk 1 above)
Comment (harias1921) [reply openapi/invoicing_v2.json:5]: Yes I love it please keep credentials up to date

**Review by harias1921 — COMMENTED** [review]
Team is doing amazing and I really need to get more sales traffic

**Pull request merged**